addons/stock/models/product_strategy.py

```diff
@@ -157,3 +144,21 @@ def _get_putaway_location(self, product, quantity=0, package=None, packaging=Non
                 checked_locations.add(location)
 
         return None
+
+
+class ProductCategory(models.Model):
```

Comment:
There is already the category class in `stock/models/product.py`

https://github.com/odoo/odoo/blob/4584a27c72f61f9e94bc66f2d0643e1d58074579/addons/stock/models/product.py#L993)